conceptnet5/nodes.py

```diff
@@ -72,8 +72,8 @@ def topic_to_concept(language, topic):
     a disambiguation string in parentheses. Returns a concept URI that
     may be disambiguated as a noun.
 
-    >>> topic_to_concept('en', 'Township (United States)')
-    '/c/en/township/n/wp/united_states'
+        >>> topic_to_concept('en', 'Township (United States)')
+        '/c/en/township/n/wp/united_states'
     """
```

Comment:
Do these still work as doctests when they're indented? Is this how doctests are supposed to be indented?

Reply:
Yes, this still works. I indented for consistency (some were indented before), but could un-intend for consistency as well.